esp/src/Sensors.h

```diff
@@ -165,5 +167,39 @@ class MovementSensor: public Sensor{
   int pin;
 };
 
+class TimeSensor: public Sensor{
+public:
+  TimeSensor(int cycle, JsonArray& channels){
```

Comment:
int cycle parameter isn't assigned to this->cycle (I think that was your intention)